common/src/main/java/com/wynntils/screens/maps/SeaskipperDepartureBoardScreen.java

```diff
@@ -0,0 +1,404 @@
+                        (yOffset + ((destinationButtons.size() - 1) * buttonHeight)),
+                        buttonWidth,
+                        buttonHeight,
+                        selected,
+                        poi);
+
+                rowOffset += 2;
+                yOffset = (int) (destinationListY + rowOffset * currentTextureScale);
+            } else if (buttonIndex % 2 == 0) {
+                newButton = new SeaskipperDestinationButton(
+                        leftX,
+                        (yOffset + ((destinationButtons.size() - (buttonIndex / 2)) * buttonHeight)),
+                        buttonWidth,
+                        buttonHeight,
+                        selected,
+                        poi);
+            } else {
+                newButton = new SeaskipperDestinationButton(
+                        (rightX + buttonWidth),
+                        (yOffset + ((destinationButtons.size() - (buttonIndex / 2) - 1) * buttonHeight)),
+                        buttonWidth,
+                        buttonHeight,
+                        selected,
+                        poi);
+
+                rowOffset += 2;
+                yOffset = (int) (destinationListY + rowOffset * currentTextureScale);
+            }
+
+            buttonIndex++;
+
+            destinationButtons.add(newButton);
+        }
+    }
+
+    public void travelToDestination() {
+        if (selectedDestination != null) {
+            Models.Seaskipper.purchasePass(selectedDestination.getDestination());
+        }
+    }
+
+    private void zoomToDestination(Poi destination) {
+        updateMapCenter(
+                destination.getLocation().getX(), destination.getLocation().getZ());
+
+        float zoomLevel = McUtils.mc().getWindow().getGuiScaledHeight() / ZOOM_SCALE;
+
+        setZoom(zoomLevel);
+    }
+
+    @Override
+    public boolean doMouseClicked(double mouseX, double mouseY, int button) {
+        for (GuiEventListener child : children()) {
+            if (child.isMouseOver(mouseX, mouseY)) {
+                child.mouseClicked(mouseX, mouseY, button);
+                return true;
+            }
+        }
+
+        for (SeaskipperDestinationButton destinationButton : destinationButtons) {
+            if (destinationButton.isMouseOver(mouseX, mouseY)) {
+                destinationButton.mouseClicked(mouseX, mouseY, button);
+
+                if (selectedDestination != null
+                        && selectedDestination.getDestination()
+                                == destinationButton.getDestination().getDestination()) {
+                    Models.Seaskipper.purchasePass(selectedDestination.getDestination());
+                } else {
+                    selectedDestination = destinationButton.getDestination();
+
+                    zoomToDestination(selectedDestination);
+                }
+
+                break;
+            }
+        }
+
+        return super.doMouseClicked(mouseX, mouseY, button);
```

Comment:
Why is this super call needed here? I am pretty sure we don't want to do anything the regular map does, do we?